src/hera/cron_workflow.py

```diff
@@ -1,8 +1,10 @@
 """The implementation of a Hera cron workflow for Argo-based cron workflows"""
-from datetime import datetime, timezone
+from datetime import datetime
+from datetime import timezone as tz
 from typing import Optional
 from uuid import uuid4
 
+import pytz
```

Comment:
I suggest we run a `make format` prior to merging so we get consistent formatting across the introduced changes! 🙂